docs/en/sql-reference/statements/create/user.md

```diff
@@ -40,6 +40,15 @@ There are multiple ways of user identification:
 - `IDENTIFIED WITH ssl_certificate CN 'mysite.com:user'`
 - `IDENTIFIED BY 'qwerty'`
 
+If a password is defined, by default it must meet the following complexity requirements:
```

Comment:
Looking at [the PR](https://github.com/ClickHouse/ClickHouse/pull/43719/files) that introduced password complexity rules (but unfortunately forgot to document them): it doesn't look like it introduced  mandatory PCRs to ClickHouse OSS - yet the docs here sound like that. You can also grep the source code of ClickHouse OSS, there are no predefined PCRs.

There are however PCRs in ClickHouse Cloud (check the dataplane repo).

Therefore, two suggestions:
- we should document here how PCRs can be created in general, e.g. using [the examples](https://github.com/ClickHouse/ClickHouse/pull/43719/files#diff-9b6b82b67400f8bcc5e2b13ca80c051175f7e9fd631fb84d81ce84941fddd2bd) in the tests of the original PR
- we should mention which PCRs are enforced in ClickHouse Cloud but I don't know the best place in the docs for that.

Reply:
Thanks Robert, added a new commit.